src/app/public/modules/auth-http-client/auth-interceptor.ts

```diff
@@ -81,9 +89,7 @@ export class SkyAuthInterceptor implements HttpInterceptor {
     if (auth) {
       const tokenContextArgs: SkyAuthTokenContextArgs = {};
 
-      if (permissionScope) {
-        tokenContextArgs.permissionScope = permissionScope;
-      }
+      tokenContextArgs.permissionScope = permissionScope || this.defaultPermissionScope;
```

Comment:
I would prefer to have the old behavior of not setting `permissionScope` on `tokenContextArgs` if there's no valid permission scope defined. I prefer to err on the side of not adding a property to an object rather than adding it with `undefined` in case the downstream code does something like `if ('permissionScope' in obj)`.

Reply:
Done.